.circleci/set-up-globals.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+set -ex
+
+#==================================================
+# The section would be transferable to a DOCKERFILE
+#==================================================
+
+# Update current apt packages
+apt-get update
+
+#=========================================================================
+# Commands below this line would not be transferable to a docker container
+#=========================================================================
+
+# Enable Composer parallel downloads
+composer global require -n "hirak/prestissimo:^0.3"
+
+# Install Terminus
+if [ ! -d $HOME/terminus ]
+then
+	# Clone terminus if it doesn't exist
+	echo -e "Installing Terminus...\n"
+	git clone --branch master git://github.com/pantheon-systems/terminus.git $HOME/terminus
```

Comment:
Why isn't this `composer global require pantheon-systems/terminus`? The exec would be in `~/.composer/vendor/bin/terminus`.